sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -54,6 +54,8 @@ static int ref_puback = 0;
 static int ref_recv = 0;
 static int ref_disconnect = 0;
 
+static int expect_msg_properties = 0;
```

Comment:
bool instead of int? Or change checks to `if (expect_msg_properties != 0)` or something like that

Reply:
Changed check to `!=0`